Update README.md (#519)
raspberrypi/pico-examples: README.md

# Raspberry Pi Pico SDK Examples

> By default, the Pico SDK targets builds for RP2040 (PICO_PLATFORM=rp2040). To build for RP2350 instead, pass

Reviewer: Is it worth mentioning that you can also use e.g. `-DPICO_BOARD=pico2` or `-DPICO_BOARD=pico2_w` and that will "automagically" select RP2350 for you?

# Raspberry Pi Pico SDK Examples

> The Pico SDK default continues to be to build for RP2040 (PICO_PLATFORM=rp2040), so to build for RP2350, you need to pass

Reviewer 2: ```suggestion
By default, the Pico SDK targets builds for RP2040 (PICO_PLATFORM=rp2040). To build for RP2350 instead, pass
```
